lib/ex_prompt.ex

```diff
@@ -168,4 +205,18 @@ defmodule ExPrompt do
         pw_loop(prompt)
     end
   end
+
+  defp string_prompt_formatter(prompt, ""), do: prompt
+  defp string_prompt_formatter(prompt, nil), do: prompt
+  defp string_prompt_formatter(prompt, default), do: "(#{default}) #{prompt}"
```

Comment:
Let's switch this to make it so that "default" value comes after the question, I think that's more the standard in similar libraries in other languages.
I think it would make readability more natural for the end-user as well; something like:

`What's your favorite color? (blue)` This way we also keep the same structure as with the `boolean_prompt`

Reply:
I think I put it in this way because of an example where a `\n` is appended at the end of the string prompt, making the default value appear at the following line. But I totally agree that it's more standard to show the default value at the end of the prompt.

Reply:
You're right, should we then "trim" the prompt, add the default value at the end and then append the "new line" if necessary? 🤔

Reply:
I can apply some regex to get the spaces at the end, if exists, and append back after default value.

Reply:
@bjufre , this is what I suggest:

If the user informs no spaces at the end, a single space is appended...

```
iex> ExPrompt.string("What is your name?")
What is your name? Castilho
"Castilho"
```

...else `ExPrompt` keeps the spaces informed.

```
iex> ExPrompt.string("What is your name?\n\n\s\s\s\s")
What is your name?

    Castilho
"Castilho"
```

I'm considering as spaces all matched by the character class [`[[:space:]]`](https://hexdocs.pm/elixir/1.12/Regex.html#module-character-classes).

Reply:
That seems reasonable, let's give it a try and see how it looks and works!